cmd/root.go

```diff
@@ -33,12 +34,17 @@ that can help you deploy, provision and operate infrastructure for SAP Applicati
 	var cfgFile string
 	var logLevel string
 
-	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.trento.yaml)")
-	rootCmd.PersistentFlags().StringVar(&logLevel, "log-level", "info", "then minimum severity (error, warn, info, debug) of logs to output")
+	rootCmd.PersistentFlags().
```

Comment:
Nothing about linting, but here, we could replace `StringVar` by `String` simply, as we don't really use this variables in this functions, and we get them later on with `viper.GetString` (or similars).

Reply:
Done!